Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -634,7 +674,7 @@ public void cleanWebAppsOnLinux() {
      * An issue is logged at https://github.com/Azure/azure-sdk-for-java/issues/1755 .
```

Comment:
The issue has been fixed for a long time. Consider to remove the workaround. Ask @jdneo for details.

Reply:
Will update the issue link, the issue still could repro.